05 type conv/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"reflect"
+)
+
+type counter int
+
+func main() {
+
+	var d counter
+	d = 0
```

Comment:
Why declare a variable like this then now give it value latter right below its, next time just use this:

`d := 0`